cluster-operator/src/main/java/io/strimzi/operator/cluster/model/Utils.java

```diff
@@ -71,6 +71,11 @@ public static String getJsonCorruptionBlame(DecodeException de, String config) {
         if (token.contains("Illegal unquoted character")) {
             return "JSON quotation";
         }
+        if (token.contains("Failed to decode: Unexpected character ")) {
```

Comment:
is the last space a typo ?

Reply:
Yes and no. The message contains space each time, but I agree it looks better withou space.